Secrets Store documentation (#567)
EGI-Federation/documentation: content/en/users/security/_index.md

title: Security Services
linkTitle: Security
type: docs
weight: 70
description: >
  Security services in the EGI infrastructure
---

## Using cloud services securely

Security in the EGI Federated infrastructure is the highest priority.
The following services can help you to configure and consume EGI services
while meeting your security and compliance objectives:

- The [Secrets Store](secrets-store) allows users to securely store credentials

>   or other sensitive information then easily retrieve it when needed, instead of
>   hardcoding it into code, configuration files, or environment variables.

Reviewer: ```suggestion
  , or other sensitive information, to later retrieve them instead of
  hardcoding them into code, configuration files, or environment variables.
```

Author: Disagree on the use of comma here (before "or"). Rephrased the rest.
